learn2track/models/gru_regression.py

```diff
@@ -48,13 +50,10 @@ def __init__(self, volume_manager, input_size, hidden_sizes, output_size, use_pr
         if self.predict_offset:
             assert self.use_previous_direction  # Need previous direction to predict offset.
 
+        # Do not use dropout in output layer
```

Comment:
1) I actually missed a line in `gru_msp`, the last layer shouldn't be dropped (and you're right, it is actually the last hidden layer, I'll change the comment)
2) I was wondering about this, if we should drop/zoneout the last hidden layer (dense layer) on keep it only in the recurrent layers like I did now.